app/models/custom_profile_field.rb

```diff
@@ -0,0 +1,16 @@
+class CustomProfileField < ApplicationRecord
+  include ActsAsProfileField
+
+  belongs_to :profile
+
+  validate :validate_maximum_count
+
+  private
+
+  # We allow a maximum of 3 custom profile fields per user
```

Comment:
This seems low for a max. I feel like 5 might be a little less restrictive.

Reply:
@mstruve This is the value the original implementation in #7246 had. My main goal was reimplementing what we previously had (and never used and then removed) in the context of the new generalized profiles. I have no strong preference in regards to the numbers, so since you expressed one 5 it shall be.